ui/lib/apps/ClusterInfo/components/StoreLocationTree/index.tsx

```diff
@@ -129,11 +131,14 @@ export default function StoreLocationTree({ dataSource }: IStoreLocationProps) {
         root.y0 = root.y
       }
 
+      const minHeight = document.documentElement.clientHeight - 80 - 48 * 2 // 48 = margin of cardInner
```

Comment:
I thought about this, but this logic bases on we only use this component inside this page, if we use this component on other pages, the minHeight is not correct. So I prefer to pass the height from outside to this component, how about it?

Reply:
yeah